client/lib/service-worker/service-worker.js

```diff
@@ -22,7 +23,26 @@ const queuedMessages = [];
  **/
 
 self.addEventListener( 'install', function( event ) {
-	event.waitUntil( self.skipWaiting() );
+	self.skipWaiting();
+
+	event.waitUntil(
+		caches.open( CACHE_VERSION ).then( function( cache ) {
+			// these are just guesses right now
+			return cache
+				.addAll( [
+					'/',
+					// these all need version numbers or hashes etc...
```

Comment:
they have hashes in prod, we turn them off in dev. see https://github.com/Automattic/wp-calypso/blob/master/webpack.config.js#L284